### ISSUE TITLE
Check for accessibility issues in the new UI (for WW 2.17)

### ISSUE THREAD

**Issue by taniwallach** [openwebwork/webwork2#1574]
We should make an accessibility review of the new UI in develop for obvious accessibility issues which can be fixed before a release.
I'm not a big expert, but the WAVE plugin can find some issues.
@Alex-Jordan may also have some advice as he seems very knowledgeable about the requirements and what really matters to users with disabilities.

**Comment by taniwallach** [comment]
1. The bread crumb of "open" item (ex. "Hmwk Sets Editor" in http://localhost:8080/webwork2/admin/instructor/sets/) does not meet contrast ratio requirements. This seems to also affect such "last" bread-crumb items students can see.
2. WAVE reports some other issues on the set editor page. but as they are staff-only fixing them is probably less critical.
   - ex: The red `*` after "Match on what? (separate multiple IDs with commas)" also has a contrast ratio failure.(Staff only - less critical).

**Comment by drgrice1** [comment]
Note that Firefox has a built in accessibility tool in its developer tools, and I have been using that to look for things already.  Thanks for pointing out the breadcrumb issue, I hadn't noticed that one.  Bootstrap 5 expects the breadcrumb text to be on a lighter background than the color we use here.  I will fix that, and probably the other issues you mentioned.

**Comment by pstaabp** [comment]
This is also addressed in #1603

**Comment by taniwallach** [comment]
It looks like quite a bit was done on this. Thanks @drgrice1 and @pstaabp.
I think we should close this issue.
Maybe a collection of remaining issues to address in the future can be collected in a new issue.

**Comment by pstaabp** [comment]
Since this is a WeBWorK issue, let's close this.  We can open a PG issue for accessibility instead.
